CONTRIBUTING.md

```diff
@@ -189,6 +189,46 @@ Take a look at the sections that follow to pick a route (or five) that appeal to
 
 If you ever find yourself at a loss for what to do, or in need of mentorship or advice on how to contribute to Bevy, feel free to ask in [Discord] and one of our more experienced community members will be happy to help.
 
+### Join a working group
+
+Active initiatives in Bevy are organized into temporary working groups: choosing one of those and asking how to help can be a fantastic way to get up to speed and be immediately useful.
+
+Working groups are public, open-membership groups that work together to tackle a broad-but-scoped initiative.
+The work that they do is coordinated in a forum-channel on [Discord](https://discord.gg/bevy), although they also create issues and may use project boards for tangible work that needs to be done.
+
+There are no special requirements to be a member, and no formal membership list.
+Working groups are *spaces*, not clubs.
+
+### Start a working group
+
+When tackling a complex initiative, friends and allies can make things go much more smoothly.
+
+To start a working group:
+
+1. Decide what the working group is going to focus on. This should be tightly focused and achievable!
+2. Gather at least 3 people including yourself who are willing to be in the working group.
```

Comment:
Since joining a working group would be easy and accessible, I think even 1 or 2 people should be able to start one - and more will join after if they want.

Reply:
Disagree. 3+ people is important for 1) consensus in discussions, 2) the ability to review and move forward PRs together, 3) to indicate commitment to the cause of the working group

Reply:
Yep, that's exactly why I picked that threshold.

Reply:
I agree that ideally 3+ people would be working on it, but I still don't think that we should disallow 2 people working groups, especially when joining them seems to be very easy - which means they're likely to fill up eventually. 

I'm worried that this rule is going to turn away a lot of new contributors, they've likely never sent a single message in the discord, so It's not going to be easy for them to recruit 2 more people for a working group. 

But I do understand we shouldn't be wasting SMEs' time with every new request to open a working group. So maybe this stage:
> The maintainers will briefly evaluate the proposal in consultation with the relevant SMEs and give you a thumbs up or down on whether this is something Bevy can and wants to explore right now.

would only come after 3 people have joined the group?

Reply:
I think if we want to keep it to a small number of groups, this is a good requirement to have. It increases the barrier to entry as well as the likelihood that the initiative will be followed through with (and hopefully see success). 

My guess is that because of this and other requirements for creating a group, we probably will only see a small handful of groups working at a time, which should hopefully make them easier to maintainers/SMEs to manage.

Reply:
> I'm worried that this rule is going to turn away a lot of new contributors, they've likely never sent a single message in the discord, so It's not going to be easy for them to recruit 2 more people for a working group.

I could be wrong, but it sounds like we're steering new contributors towards _joining_ working groups rather than going straight into _starting_ one (unless, of course, that's what they really want to do haha).

Reply:
Yep, precisely!